ARCore/app/src/main/java/com/novoda/spikes/arcore/poly/APIAssets.kt

```diff
@@ -0,0 +1,9 @@
+package com.novoda.spikes.arcore.poly
+
+data class APIAssets(val assets: List<APIAsset>)
```

Comment:
can these be `Api*` (same for others), so it's in sync with `PolyApi`

Reply:
Can do